### PR TITLE
feat: Add datasets/subcohorts info card to landing

### PR DESCRIPTION
**Pull request by connoratrug**
note: default label and text can be overridden via setting ( see docs )

### CODE REVIEW

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue [hunk 1]
```diff
@@ -186,7 +191,7 @@ function getSettingValue(settingKey: string, settings: ISetting[]) {
             "CATALOGUE_LANDING_SAMPLES_TEXT",
             data.data._settings
           ) ||
-          "The cumulative number of participants with samples collected of all datasets combined."
+          "The cumulative number of participants with samples collected of all datasets combined"
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue:194]: so we assume subcohorts are not overlapping?

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue [hunk 2]
```diff
@@ -212,6 +217,25 @@ function getSettingValue(settingKey: string, settings: ISetting[]) {
         }}
         cross-sectional.
       </LandingCardSecondary>
+
+      <LandingCardSecondary icon="viewTable">
+        <b>
+          {{ data.data.Subcohorts_agg.count }}
+          {{
+            getSettingValue(
+              "CATALOGUE_LANDING_DATASETS_LABEL",
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue:226]: I am confused: datasets and subcohorts are not the same thing?

**Review by mswertz — CHANGES_REQUESTED** [review]
I think something may be wrong: datasets and subcohorts are orthogonal entities. I.e. a dataset can have data on one or more subcohorts.

**Comment by connoratrug** [comment]
@mswertz suggest you discuss this with DataManager ( Kim )

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue (hunk 1 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue:194]: i guess so, the maintainer can override the default/fallback text

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue (hunk 2 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue]: see additional commits

**Review by mswertz — CHANGES_REQUESTED** [review]

File: docs/catalogue/cat_server_side-rendered.md [hunk 3]
```diff
@@ -85,6 +85,10 @@ The label shown on landing CTA element for each of the main sections
 
 ```CATALOGUE_LANDING_DESIGN_TEXT```
 
+```CATALOGUE_LANDING_DATASETS_LABEL```
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/catalogue/cat_server_side-rendered.md:88]: a yes, forgot to update that

[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/catalogue/cat_server_side-rendered.md]: done

**Review by mswertz — APPROVED** [review]

**Pull request merged**